js/build_map.js

```diff
@@ -134,32 +163,70 @@ function fillMap() {
+    waterUseViz.dims.svg.height = waterUseViz.dims.map.height;
+    // watermark is at bottom left
+    waterUseViz.dims.watermark.x0 = waterUseViz.dims.svg.width * 0.01;
+    waterUseViz.dims.watermark.y0 = waterUseViz.dims.svg.height * 0.95;
     
-  d3.select('#button-container')
-    .selectAll('button')
-    .data(categories)
-    .enter()
-    .append('button')
-    .text(function(d){
-      return categoryToName(d);
-    })
-    .attr('class', function(d){
-      return d;
-    })
-    .on('click', function(d){
-      updateCategory(d.toLowerCase(), activeCategory);
-    })
-    .on('mouseover', function(d){
-      showCategory(d.toLowerCase(), activeCategory, action = 'mouseover');
-    })
-    .on('mouseout', function(d){
-      showCategory(activeCategory, d.toLowerCase(), action = 'mouseout');
-    });
+  } else {
+  
+    // buttonBox sits below map with small vertical buffer between map and buttons
+    waterUseViz.dims.buttonBox.y0 = waterUseViz.dims.map.height * 1.05;
+    waterUseViz.dims.buttonBox.width = waterUseViz.dims.map.width;
+    waterUseViz.dims.buttonBox.height = waterUseViz.dims.buttonBox.width *
+      (waterUseViz.dims.buttonBox.heightDesktop / waterUseViz.dims.buttonBox.widthDesktop);
+    // map fills the top part of the svg
+    waterUseViz.dims.map.x0 = 0;
+    // svg is [map]
+    //        [buttons]
+    waterUseViz.dims.svg.width = waterUseViz.dims.map.width;
+    waterUseViz.dims.svg.height = waterUseViz.dims.buttonBox.y0 + waterUseViz.dims.buttonBox.height;
+    // watermark is at bottom right
+    waterUseViz.dims.watermark.x0 = waterUseViz.dims.svg.width * 0.85;
+    waterUseViz.dims.watermark.y0 = waterUseViz.dims.svg.height * 0.95;
+  }
+  
+  // Apply the changes to the svg, map, map background, and watermark
+  svg
+    .attr('viewBox', '0 0 ' + waterUseViz.dims.svg.width + ' ' + waterUseViz.dims.svg.height + '');
+  map
+    .attr('transform', 'translate(' + waterUseViz.dims.map.x0 + ', ' + 0 + ')');
+  mapBackground
+    .attr("width", waterUseViz.dims.svg.width)
+    .attr("height", waterUseViz.dims.map.height);
+  watermark
+    .attr('transform', 'translate(' + waterUseViz.dims.watermark.x0 + ',' + waterUseViz.dims.watermark.x0 + ')scale(0.25)');
+  
+  // Apply the changes to the button elements
+  waterUseViz.elements.buttonBox
+    .attr('transform', 'translate(' + 0 + ', ' + waterUseViz.dims.buttonBox.y0 + ')');
+  waterUseViz.elements.buttonBox.select('#button-background')  
+    .attr('width', waterUseViz.dims.buttonBox.width);
+  waterUseViz.elements.buttonBox.selectAll('.button .category-label')
+    .attr('x', waterUseViz.dims.buttonBox.width * 0.05); // nudge a little over from the rectangle's left edge
+  waterUseViz.elements.buttonBox.selectAll('.button .category-amount')
+    .attr('x', waterUseViz.dims.buttonBox.width * 0.9); // nudge a little over from the rectangle's left edge
+  updateButtons(activeCategory); // updates the button rectangle widths
```

Comment:
`activeCategory` would also be one of those global things, right?

Reply:
yes, and see above - saving that for issue #127.